boards.txt

```diff
@@ -2796,6 +2796,14 @@ GenF7.build.flags.fp=-mfpu=fpv4-sp-d16 -mfloat-abi=hard
 GenF7.build.series=STM32F7xx
 GenF7.build.cmsis_lib_gcc=arm_cortexM7lfsp_math
 
+# Generic F722RETx
```

Comment:
I would add also the 3 other mcu's here:
https://github.com/stm32duino/Arduino_Core_STM32/blob/main/variants/STM32F7xx/F722R(C-E)T_F730R8T_F732RET/boards_entry.txt